.circleci/config.yml

```diff
@@ -0,0 +1,185 @@
+- &save-gradle-cache
+  paths:
+  - ~/.gradle
+  key: gradle-cache-{{ checksum "android/build.gradle" }}-{{ checksum "example/android/build.gradle" }}-{{ checksum "example/android/app/build.gradle" }}
+
+# INSTALLATION
+- &yarn
+  name: Yarn Install
+  command: |
+    yarn install --network-concurrency 1 --non-interactive --cache-folder ~/.cache/yarn & wait
+
+# ANALYSE
+- &eslint
+  name: ESLint Checks
+  command: yarn test:eslint
+
+- &flow
+  name: Flow Checks
+  command: yarn test:flow
+
+- &jest
+  name: Jest Unit Tests
+  command: yarn test:jest
+
+# -------------------------
+#          JOBS
+# -------------------------
+version: 2
+jobs:
+  # Set up a Linux environment for downstream jobs
+  linux-checkout:
+    <<: *linux_defaults
+    steps:
+    - checkout
+    - restore-cache: *restore-yarn-cache
+    - run: rm -rf node_modules
+    - run: yarn cache clean
+    - run: *yarn
+    - save-cache: *save-yarn-cache
+    - persist_to_workspace:
+        root: .
+        paths: .
+
+  eslint:
+    <<: *linux_defaults
+    steps:
+    - attach_workspace:
+        at: ~/react-native-geolocation
+    - run: *eslint
+
+  flow:
+    <<: *linux_defaults
+    steps:
+    - attach_workspace:
+        at: ~/react-native-geolocation
+    - run: *flow
+  
+  jest:
+    <<: *linux_defaults
+    steps:
+    - attach_workspace:
+        at: ~/react-native-geolocation
+    - run: *jest
+  
+  android-compile:
+    <<: *android_defaults
+    steps:
+    - attach_workspace:
+        at: ~/react-native-geolocation
+    - restore-cache: *restore-gradle-cache
+    - run:
+        name: Accept Android licences
+        command: |-
+          yes | sdkmanager --licenses || exit 0
+          yes | sdkmanager --update || exit 0
+    - run:
+        name: Build Android Example App and Library
+        command: |-
+          cd example/android
+          ./gradlew clean assembleDebug
+    - save-cache: *save-gradle-cache
+  
+  ios-checkout:
+    <<: *macos_defaults
+    steps:
+    - checkout
+    - restore-cache: *restore-yarn-cache
+    - run: rm -rf node_modules
+    - run: yarn cache clean
+    - run: *yarn
+    - save-cache: *save-yarn-cache
+    - persist_to_workspace:
+        root: .
+        paths: .
+
+  ios-compile:
+    <<: *macos_defaults
+    steps:
+    - attach_workspace:
+        at: ~/react-native-geolocation
+    - run:
+        name: Build example app
+        command: |-
+          react-native run-ios --project-path example/ios
+
+# -------------------------
+#        WORKFLOWS
+# -------------------------
+workflows:
+  version: 2
+  Test:
+    jobs:
+    - linux-checkout
+    - eslint:
+        requires:
+        - linux-checkout
+    - flow:
+        requires:
+        - linux-checkout
+    - jest:
+        requires:
+        - linux-checkout
+    - android-compile:
+        requires:
+        - linux-checkout
+# Disabled until we have macOS containers enabled        
```

Comment:
I think macOS is available now, do you mind enabling them?

Reply:
Sure